.github/workflows/main.yml

```
@@ -18,6 +18,12 @@ jobs:
         python-version: [ 3.6, 3.7 ]
         torch-version: [ 1.5.0]
         tensorflow-version: [ 1.15.0 ]
+        test-details:
+          - {dep: allennlp, testfile: tests/allennlp_processors_test.py}
+          - {dep: nltk, testfile: tests/nltk_processors_test.py}
+          - {dep: "varder,nltk", testfile: tests/sentiment_processor_test.py}
+          - {dep: spacy, testile: tests/spacy_processors_test.py}
```

the failure is because of the typo here, 
```suggestion
          - {dep: spacy, testfile: tests/spacy_processors_test.py}
```